apps/src/lib/node/ledger/ethereum_node/events.rs

```diff
@@ -202,7 +202,7 @@ pub mod eth_events {
         /// Check if the minimum number of confirmations has been
         /// reached at the input block height.
         pub fn is_confirmed(&self, height: &Uint256) -> bool {
```

Comment:
We should add a test to cover the issue that this PR fixes i.e. asserting that if the block height difference is greater than `self.confirmations`, then `is_confirmed` does return `true`

Reply:
I agree.

Reply:
Actually, we do have a test that catches this bug. The `test_oracle::test_finality_gadget`. It caught a few more bugs even. Fixing now.